cmd/up/up.go

```diff
@@ -503,27 +506,38 @@ func (up *upContext) shutdown() {
 
 }
 
-func printDisplayContext(dev *model.Dev) {
+func printDisplayContext(ctx context.Context, dev *model.Dev, c kubernetes.Interface) {
 	if dev.Context != "" {
-		log.Println(fmt.Sprintf("    %s   %s", log.BlueString("Context:"), dev.Context))
+		log.Println(fmt.Sprintf("    %s    %s", log.BlueString("Context:"), dev.Context))
 	}
-	log.Println(fmt.Sprintf("    %s %s", log.BlueString("Namespace:"), dev.Namespace))
-	log.Println(fmt.Sprintf("    %s      %s", log.BlueString("Name:"), dev.Name))
+	log.Println(fmt.Sprintf("    %s  %s", log.BlueString("Namespace:"), dev.Namespace))
+	log.Println(fmt.Sprintf("    %s       %s", log.BlueString("Name:"), dev.Name))
 
 	if len(dev.Forward) > 0 {
 		for i := 0; i < len(dev.Forward); i++ {
 			if dev.Forward[i].Service {
-				log.Println(fmt.Sprintf("               %d -> %s:%d", dev.Forward[i].Local, dev.Forward[i].ServiceName, dev.Forward[i].Remote))
+				log.Println(fmt.Sprintf("                %d -> %s:%d", dev.Forward[i].Local, dev.Forward[i].ServiceName, dev.Forward[i].Remote))
 				continue
 			}
-			log.Println(fmt.Sprintf("               %d -> %d", dev.Forward[i].Local, dev.Forward[i].Remote))
+			log.Println(fmt.Sprintf("                %d -> %d", dev.Forward[i].Local, dev.Forward[i].Remote))
 		}
 	}
 
 	if len(dev.Reverse) > 0 {
-		log.Println(fmt.Sprintf("    %s   %d <- %d", log.BlueString("Reverse:"), dev.Reverse[0].Local, dev.Reverse[0].Remote))
+		log.Println(fmt.Sprintf("    %s      %d <- %d", log.BlueString("Reverse:"), dev.Reverse[0].Local, dev.Reverse[0].Remote))
 		for i := 1; i < len(dev.Reverse); i++ {
-			log.Println(fmt.Sprintf("               %d <- %d", dev.Reverse[i].Local, dev.Reverse[i].Remote))
+			log.Println(fmt.Sprintf("                %d <- %d", dev.Reverse[i].Local, dev.Reverse[i].Remote))
+		}
+	}
+
+	if dev.Divert != nil {
+		username := okteto.GetSanitizedUsername()
+		name := diverts.DivertName(username, dev.Divert.Ingress)
+		i, err := ingressesv1.Get(ctx, name, dev.Namespace, c)
+		if err != nil {
+			log.Infof("error getting diverted ingress %s: %s", name, err.Error())
+		} else if len(i.Spec.Rules) > 0 {
+			log.Println(fmt.Sprintf("    %s %s", log.BlueString("Divert URL:"), i.Spec.Rules[0].Host))
```

Comment:
I don't think we should use the name `Divert` in the messages,  since it's not a name that tells the user anything about the feature or the usage. Could we instead just say it's the application or deve environment URL?